aglio-olio.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html> 
+<html>
+    <head>
+        <title>Aglio Olio</title>
+        <link rel="stylesheet" href="individual-styles.css">
+    </head>
+    <body>
+        <div class="main-photo">
+                      <img src="aglio-olio.jpg" alt="aglio olio" width="410">
+        <div class="content">
+               <h1>Aglio Olio</h1>
+                 <p>Preparation time: 25mins</p>
+                 <p>Cook: 3 hrs and 30 mins</p>
+        </div>
+    </div>
+        <fieldset>
```

Comment:
Why did you use the fieldset tag here? This tag usually is used for forms

Reply:
I see! hmm, would you advise that we separate structure and design for HTML elements? I included fieldset and legend elements only because I thought they look nice. I didn't think much about the appropriate usage of the elements.

Reply:
Hmm by right most tag will not have a design on its own. They would look nice if you style them accordingly with CSS. Means, you could replace fieldset and legend easily with a div or the like. Could you go in details what you mean by they look nice?

Yup, we should always try to use semantically correct HTML. That improves search engine optimization and will make it easier for screen readers to interpret what the page displays.